content/spin/v2/manifest-reference.md

```diff
@@ -129,7 +129,7 @@ Each array entry contains a mix of common fields and trigger-specific fields.
 
 ## The `component` Table
 
-The keys of the `component` table, usually written as part of the table syntax e.g. `[component.my-component]`, are user-defined.  (In the preceding example, the key is `my-component`). Component names must be kebab-cased, i.e. the only permitted separator is a hyphen.
+The keys of the `component` table, usually written using TOML table syntax e.g. `[component.my-component]`, are ids of components.  Component ids must be kebab-cased, i.e. the only permitted separator is a hyphen.  In the following example, `my-component` is used as an example component id.
```

Comment:
I find "using TOML table syntax" less specific than "as part of the table syntax", because the latter guides me more to it being a _part_ of something rather than the usual key-value table syntax.  But neither is ideal.  We can and should wordsmith this, but perhaps it would be better to decouple that from the "missing component name in the variables example" fix as I can see a _lot_ of tweaking being needed to get this right, whereas the fix we can just knock off.